libs/utils/tradeHistoryAPI.ts

```diff
@@ -44,7 +44,7 @@ export type TradeHistory = {
 const fetchTradeHistory: (params: {
     network?: NetworkType;
     account: string;
-    type: 'mint' | 'burn';
+    type: 'mint' | 'burn' | 'flip';
```

Comment:
I think it will also need some handling further down in this function, you might have to ask Chris what the params should be

Reply:
@ScaredIbis FYI